test/compress/new.js

```diff
@@ -11,10 +11,42 @@ new_statement: {
     expect_exact: "new x(1);new x(1)(2);new x(1)(2)(3);new new x(1);new new x(1)(2);new new x(1)(2);(new new x(1))(2);"
 }
 
+new_statements_2: {
+    input: {
+        new x;
+        new new x;
```

Comment:
This compresses to `new(new x)`, but those parens are superfluous? (same for some following cases)